### DISCUSSION TITLE
Query about submission

### DISCUSSION THREAD

**Discussion by imsandeepk** [DiveHQ/backend-internship-task#25] [Q&A] [answered]
We just have to push our code to the private repo we have been added by GitHub classroom, we don't have to make a PR or something right?

**Comment by goofynugtz** [comment]
I have the same doubt.
Also to mention, I can see the private repo in my browser but I cannot clone or fork it. 
When I try to clone it using ssh, It gives me an error "Repository not found" and I do not have correct access rights.
PS: I can fork my other private repos or even other organizations through ssh.

Any help with how do I make submission?

**Comment by sshsrn** [comment, marked as answer]
> We just have to push our code to the private repo we have been added by GitHub classroom, we don't have to make a PR or something right?

From the contents of my task submission repo, I think we have to create a new branch, upload our code there and then create a PR for submission.
![image](https://github.com/DiveHQ/backend-internship-task/assets/62781914/42d1762f-4b7b-40e9-b0de-180d05ff2f2e)

**Comment by shuklaritvik06** [comment]
> Any help with how do I make submission?

Make  a branch from main branch of your private repo, do the changes there and create PR from that branch to main and merge that PR (typical github flow) and now the changes would be visible in the feedback PR already created by github classroom this will be used for feedback

**Reply by imsandeepk** [reply]
I was already pushing the code to the main branch, so now what should I do , should I make a PR to feedback?

**Comment by shuklaritvik06** [comment]
> I was already pushing the code to the main branch, so now what should I do , should I make a PR to feedback?

No No It's Ok Changes must have been updated in the Feedback PR please check

**Reply by 0xkarambit** [reply]
do we need to close the PR now that its time ? or will it be automatically handled by github-classroom ?

**Reply by shuklaritvik06** [reply]
You do not have to close the Feedback#1 PR till the mentors ask you to do so
